src/libraries/System.Console/tests/ReadAndWrite.cs

```diff
@@ -280,6 +281,14 @@ public static unsafe void OutputEncodingPreamble()
         }
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.Browser)]
+    public static unsafe void OutputEncodingPreamble_Browser()
```

Comment:
nit: doesn't need unsafe:

```suggestion
    public static void OutputEncodingPreamble_Browser()
```

Reply:
fixed